test/interfaces/test_midi.py

```diff
@@ -2,14 +2,14 @@
 import unittest
 import unittest.mock
 
-import mido
-import shc.interfaces.midi
-from shc.datatypes import RangeUInt8
-
 from .._helper import InterfaceThreadRunner
 
 
 try:
+    import mido
+    import shc.interfaces.midi
+    from shc.datatypes import RangeUInt8
+
```

Comment:
Needed to move the import here.

Avoids the `pip uninstall python-rtmidi` to skip those tests.  Not sure whether this is really the best logic as this happens quietly and might not be noticed if skipped accidently.